beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -192,6 +194,13 @@ func (s *Service) processAndBroadcastBlock(ctx context.Context, b interfaces.Rea
 		}
 	}
 
+	bestPeers := s.getBestPeers()
+	if len(bestPeers) > 0 {
+		if err := s.requestPendingBlobs(ctx, b.Block(), blkRoot, bestPeers[rand.NewGenerator().Int()%len(s.getBestPeers())]); err != nil {
```

Comment:
why is it calling getBestPeers inside again? is this supposed to do that? and not just `len(bestPeers)`
also with the modulo in the middle this becomes less readable can you break it out into it's own variable with a comment why we are doing this?